The job **`tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/27262580778/job/80511953360) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/80511953360) [(plain)](https://github.com/rust-lang/rust/commit/61aa16370b53d07c397812040210aede961ef34d/checks/80511953360/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
fmt check
fmt: checked 6895 files
Diff in /checkout/library/core/src/task/wake.rs:434:
     ///
     /// To avoid missed wakeups, runtimes must ensure that for any call to `wake`,
     /// there is a subsequent call to `poll` such that the call to `wake()` _happens-before_
-    /// the beginning of the invocation of `poll`. 
+    /// the beginning of the invocation of `poll`.
     /// In particular, this means that if a task self-wakes (invokes `wake` on itself during `poll`),
     /// then the `poll` must be invoked again because the call to `wake` _happens-after_ the beginning
     /// of the current invocation of `poll`.
Bootstrap failed while executing `test src/tools/tidy tidyselftest --extra-checks=py,cpp,js,spellcheck`
Build completed unsuccessfully in 0:00:48
  local time: Wed Jun 10 08:16:18 UTC 2026
  network time: Wed, 10 Jun 2026 08:16:18 GMT
##[error]Process completed with exit code 1.
```

</details>

--- a/library/alloc/src/task.rs
+++ b/library/alloc/src/task.rs
@@ -39,7 +39,7 @@ use crate::sync::Arc;
 ///
 /// # Memory Ordering
 ///
-/// To avoid missed wakeups, all runtimes must adhere to the requirement described for [`Waker::wake`].
+/// To avoid missed wakeups, all executors must adhere to the requirement described for [`Waker::wake`].
 ///
 /// # Examples
 ///

--- a/library/core/src/task/wake.rs
+++ b/library/core/src/task/wake.rs
@@ -434,7 +434,7 @@ impl Waker {
     ///
     /// To avoid missed wakeups, runtimes must ensure that for any call to `wake`,
     /// there is a subsequent call to `poll` such that the call to `wake()` _happens-before_
-    /// the beginning of the invocation of `poll`. 
+    /// the beginning of the invocation of `poll`.
     /// In particular, this means that if a task self-wakes (invokes `wake` on itself during `poll`),
     /// then the `poll` must be invoked again because the call to `wake` _happens-after_ the beginning
     /// of the current invocation of `poll`.

--- a/library/core/src/task/wake.rs
+++ b/library/core/src/task/wake.rs
@@ -432,12 +432,13 @@ impl Waker {
     /// executor’s choice which task to run and the executor may choose to run the
     /// current task again.
     ///
-    /// To avoid missed wakeups, runtimes must ensure that for any call to `wake`,
-    /// there is a subsequent call to `poll` such that the call to `wake()` _happens-before_
-    /// the beginning of the invocation of `poll`.
-    /// In particular, this means that if a task self-wakes (invokes `wake` on itself during `poll`),
-    /// then the `poll` must be invoked again because the call to `wake` _happens-after_ the beginning
-    /// of the current invocation of `poll`.
+    /// To avoid missed wakeups, runtimes must ensure that for any call to
+    /// `wake`, there is a subsequent call to `poll` such that the call to
+    /// `wake()` _happens-before_ the beginning of the invocation of `poll`. In
+    /// particular, this means that if a task self-wakes (invokes `wake` on
+    /// itself during `poll`), then the `poll` must be invoked again because the
+    /// call to `wake` _happens-after_ the beginning of the current invocation
+    /// of `poll`.
     ///
     /// [`poll()`]: crate::future::Future::poll
     #[inline]

